### PR TITLE
Suzuka

### PR DESCRIPTION
**Pull request by 0xPrimata**
merge suzuka into main to branch off main for devnet and inola

### CODE REVIEW

File: src/global-config/network-selection.ts [hunk 1]
```diff
@@ -16,17 +16,17 @@ function getUserSelectedNetworkFromLocalStorageWithDefault(): NetworkName {
...
-    SELECTED_NETWORK_LOCAL_STORAGE_KEY,
-  );
-  if (network === defaultNetworkName && currentLocalStorageNetwork != null) {
-    // if network selection is default network (i.e. mainnet) we remove the local storage entry
-    localStorage.removeItem(SELECTED_NETWORK_LOCAL_STORAGE_KEY);
-  } else if (currentLocalStorageNetwork !== network) {
-    localStorage.setItem(SELECTED_NETWORK_LOCAL_STORAGE_KEY, network);
-  }
-}
+// function writeSelectedNetworkToLocalStorage(network: NetworkName) {
```
Comment (andyjsbell) [inline src/global-config/network-selection.ts:19]: Is this something we can remove?

File: src/global-config/network-selection.ts [hunk 2]
```diff
@@ -39,23 +39,25 @@ function writeSelectedNetworkToLocalStorage(network: NetworkName) {
...
-    setSearchParams(
-      (prev) => {
-        const newParams = new URLSearchParams(prev);
-        newParams.set("network", network);
-        return newParams;
-      },
-      {replace},
-    );
-    writeSelectedNetworkToLocalStorage(network);
+    console.log(replace);
```
Comment (andyjsbell) [inline src/global-config/network-selection.ts:50]: ditto

File: src/pages/Block/Index.tsx [hunk 3]
```diff
@@ -9,6 +9,7 @@ import PageHeader from "../layout/PageHeader";
 
 export default function BlockPage() {
   const {height} = useParams();
+  console.log("height", height);
```
Comment (andyjsbell) [inline src/pages/Block/Index.tsx:12]: would be good to remove this

**Review by andyjsbell — COMMENTED** [review]
a couple of nits but lgtm

File: src/global-config/network-selection.ts (hunk 1 above)
Comment (0xPrimata) [reply src/global-config/network-selection.ts:19]: wouldn't remove it for now

**Pull request merged**